server/src/controllers/finance.controller.js

```diff
@@ -72,8 +77,9 @@ module.exports.addTransaction = async (req, res) => {
         description: req.body.description,
         currency: 'HUF',
         pocket: req.body.pocket,
-        entryDate: date,
-        dueDate: date
+        entryDate: dateNow,
+        dueDate: dueDateToDB,
+        by: req.user.email
```

Comment:
Are we going to display the "by" property somewhere on the UI, or is this just for emergencies so that we can look it up in the DB if necessary?

Reply:
Just in the DB.